### PR TITLE
Add Bin class

### PR DESCRIPTION
**Pull request by FarzanRashid**

### CODE REVIEW

File: tests/test_bin.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+class TestBin(TestCase):
+    def setUp(self):
+        self.oc1 = Outcome("Red", 1)
+        self.oc2 = Outcome("Black", 2)
+
+    def test_bin_objects_can_be_created_with_outcome_objects(self):
+        b1 = Bin([self.oc1, self.oc2])
+        b2 = Bin([self.oc2, self.oc2])
+        self.assertEqual(isinstance(b1, Bin), True)
+        self.assertEqual(isinstance(b2, Bin), True)
```
Comment (FarsanRashid) [inline tests/test_bin.py:14]: better use `self.assertTrue`

File: tests/test_bin.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from roulette import Bin, Outcome
+from unittest import TestCase
```
Comment (FarsanRashid) [inline tests/test_bin.py:2]: standard libs should come first in import order

File: tests/test_bin.py (hunk 1 above)
Comment (FarzanRashid) [reply tests/test_bin.py]: Done. Thanks.

File: tests/test_bin.py (hunk 2 above)
Comment (FarzanRashid) [reply tests/test_bin.py]: Done. Thanks.

**Review by FarsanRashid — APPROVED** [review]

**Pull request merged**